Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -153,8 +162,35 @@ private void RoundEnd(RoundEndMessageEvent message)
             if (_window?.RoundId == message.RoundId)
                 return;
 
-            //This is not ideal at all, but I don't see an immediately better fit anywhere else.
-            _window = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText, message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, _entityManager);
+            //Cyberfinn changes:
+            //back up the round info:
+            _roundEndContainer._message = message;
+            //display the window
+            DisplayRoundEndSummary(_roundEndContainer._message);
+
+            //enable the button, so that users can review the summary after closing the window
+            EnableRoundEndSummaryButton();
+        }
+        //Cyberfinn changes: moved this out to its own method
+        public static void DisplayRoundEndSummary(RoundEndMessageEvent? message)
```

Comment:
this should not be static same with _entityManager

Reply:
made static so that I can access the same instance of the object from other parts of the code, didnt want to spawn a new gameTicking manager, in fear that it would bug something out or cause memory issues. was safer for me to use the same object across different parts of the app using static. Also, this is my first time looking at the SS14 code - and I dont know anything about game dev, so just using what I know to get stuff to work lol